clarification on IP address for listener

chunsiongtan opened the pull request:
I've used this guide multiple times, and I have 2 instances of customers giving feedback.
The listener IP address could be explicitly stated, that is has to be different from cluster IP address.

azurecla commented:
Hi __@anzaman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (alzam). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

anzaman commented:
@ganesr @cherylmc @charwen

chunsiongtan committed:
clarification on IP address for listener

I've used this guide multiple times, and I have 2 instances of customers giving feedback.
The listener IP address could be explicitly stated, that is has to be different from cluster IP address.

MikeRayMSFT reviewed (approved):
@chunsiongtan thank you for your submission.
